cli/info/info.go

```diff
@@ -163,6 +185,10 @@ func NewCommand() *cli.Command {
 		Description: "With nknc info, you could look up blocks, transactions, etc.",
 		ArgsUsage:   "[args]",
 		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "pretty, p",
```

Comment:
-p is also used by nknc for password

Reply:
It just is flag of subCommand. `info -p` NOT same cli.Command.Action with `wallet -p`